yurt/yurt_core/env.py

```diff
@@ -0,0 +1,114 @@
+# coding=utf-8
+
+import click
+import os
+import re
+import yaml
+from cookiecutter.main import cookiecutter
+from shutil import copy2, make_archive
+from yurt.yurt_core.utils import get_iter, find_file_in_ancestor
+from zipfile import ZipFile
+
+
+# Helpers
+
+def in_yurt_directory():
+    return os.path.exists('./docker-compose.yml') and \
+           os.path.exists('./django_app') and \
+           os.path.exists('./envs')
```

Comment:
Is it worth making some of these re-used strings / paths variables? Not 100% sure.